crates/ruff_python_parser/tests/snapshots/invalid_syntax@expressions__compare__missing_lhs.py.snap

```diff
@@ -0,0 +1,62 @@
+---
+source: crates/ruff_python_parser/tests/fixtures.rs
+input_file: crates/ruff_python_parser/resources/invalid/expressions/compare/missing_lhs.py
+---
+## AST
+
+```
+Module(
+    ModModule {
+        range: 0..10,
+        body: [
+            Expr(
+                StmtExpr {
+                    range: 2..3,
+                    value: Name(
+                        ExprName {
+                            range: 2..3,
+                            id: "y",
+                            ctx: Load,
+                        },
+                    ),
+                },
+            ),
+            Expr(
```

Comment:
The parse tree here looks incorrect. What happened with the `>` token? I think this should parse as a comparator expression with a missing left hand side. 

Edit: Okay, I think what's happening here is that we drop the `>` as part of the statement parsing because we aren't at the start of an expression. That's fine. We can do the other improvement later.